lib/archethic_web/router.ex

```diff
@@ -26,6 +26,11 @@ defmodule ArchEthicWeb.Router do
     get("/up", UpController, :up)
     get("/metrics", MetricsController, :index)
     live_dashboard("/dashboard", metrics: ArchEthic.Telemetry)
+
+    if Mix.env() in [:dev, :test] or System.get_env("ARCHETHIC_NETWORK_TYPE") == "testnet" do
```

Comment:
Will this `Mix.env() in [:dev, :test]` work with release ?

Reply:
Otherwise, you can configuration

Reply:
Yes but Mix.env() will be called

Reply:
Okay I get it.

Reply:
Maybe using Application.get_env should do it, the System.get_env could loaded from the conf

Reply:
What if I use `cond`?

Reply:
Cond is use for multiple condition like switch

Reply:
For example :

```elixir
if System.get_env("ARCHETHIC_NETWORK_TYPE") == "testnet" do
  do_something()
else
  if Mix.env() in [:dev, :test] do
    do_something()
  end
end
```

Will this logic work?

Reply:
For example :

```elixir
if System.get_env("ARCHETHIC_NETWORK_TYPE") == "testnet" do
  do_something()
else
  if System.get_env("ARCHETHIC_NETWORK_TYPE") != "mainnet" and Mix.env() in [:dev, :test] do
    do_something()
  end
end
```

Will this logic work?

Reply:
You can avoid all of this complexity by just using conf .

Reply:
But is conf executed in release?

Reply:
Yes sure . Compilation or runtime configuration

Reply:
It's the cleanest way to deal with env variable

Reply:
So to keep testnet and mainnet separate in prod, one needs to change the config?

Reply:
I think it's the idea so we could spawn Mainnet and testnet node just by specifying this conf and of course the bootstrapping seeds (but this is not on this PR subject)